docker-compose.yml

```diff
@@ -7,13 +7,5 @@ services:
           - mojaloop/quoting-service
           - quoting-service
       container_name: qs_quoting-service
-      links:
-        - mysql
-        - kafka
```

Comment:
Hey Steven why did you remove this?

Reply:
@rmothilal I removed them because `kafka` and `mysql` services are not defined within that `docker-compose.yml` file.   So linking to them doesn't seem necessary.